<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>772</width>
    <height>365</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #f5c790;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>631</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Bookman L</family>
      <pointsize>20</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>N REINAS</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>90</y>
      <width>631</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Bookman L</family>
      <pointsize>20</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #c48d6e;</string>
    </property>
    <property name="text">
     <string>DIFFERENTIAL EVOLUTION</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Bookman L</family>
      <pointsize>12</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Numero de &lt;/p&gt;&lt;p align=&quot;center&quot;&gt;Reinas&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QComboBox" name="QueenBox">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>270</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="StartButton">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>280</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Bookman L</family>
      <pointsize>16</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>Â¡EMPEZAR!</string>
    </property>
   </widget>
   <widget class="QComboBox" name="IterBox">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>170</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>111</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Bookman L</family>
      <pointsize>12</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Numero de&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;Iteraciones&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QComboBox" name="IndBox">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>170</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>170</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Bookman L</family>
      <pointsize>12</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Numero de &lt;/p&gt;&lt;p align=&quot;center&quot;&gt;individuos&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
